v2/common/src/main/java/com/google/cloud/teleport/v2/transforms/FileFormatFactory.java

```diff
@@ -0,0 +1,124 @@
+/*
```

Comment:
I am not sure if this should be in common. This should be in the kafka-to-gcs module.